spec/unit/type/ec2_securitygroup_spec.rb

```diff
@@ -31,4 +31,14 @@
       expect(type_class.parameters).to be_include(param)
     end
   end
+
+  it 'should order tags on output' do
```

Comment:
Me and @zreichert had been talking about moving a few things into customer rspec matchers and the like. Agree this is a prime candidate.